common/djangoapps/student/models.py

```diff
@@ -929,12 +929,15 @@ def is_course_full(self, course):
 
         return is_course_full
 
-    def users_enrolled_in(self, course_id):
-        """Return a queryset of User for every user enrolled in the course."""
-        return User.objects.filter(
-            courseenrollment__course_id=course_id,
-            courseenrollment__is_active=True
-        )
+    def users_enrolled_in(self, course_id, is_active=True):
+        """Return a queryset of User for every user enrolled in the course.  If is_active=None, returns both
```

Comment:
What do you think about having `is_active` truly be a `boolean` value and not be a combination of both boolean (via `True`) and `None`?  As a developer who sees this method signature, I would assume that I could pass in a value of `False` to a parameter defaulted to `True`.